index.html

```diff
@@ -0,0 +1,1547 @@
+                </div>
+                <div class="my-turn-div p-sm-0">
+                  <img class="my-turn-img mb-2 w-100" src="./images/love-coronavirus-immigration.jpg" alt="" />
+                  <div>
+                    <p>
+                      'My Fiancé and I Are Stuck 3,400 Miles Apart Indefinitely'
+                    </p>
+                  </div>
+                </div>
+                <div class="my-turn-div p-sm-0">
+                  <img class="my-turn-img mb-2 w-100" src="./images/love-coronavirus-immigration.jpg" alt="" />
+                  <div>
+                    <p>
+                      'My Fiancé and I Are Stuck 3,400 Miles Apart Indefinitely'
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <!-- culture and travel -->
+            <div class="order-sm-5 my-turn bg-white pt-3 mt-3 container-fluid d-sm-none d-md-block">
+              <h3 class="CULTURE-AND-TRAVEL">CULTURE AND TRAVEL</h3>
+
+              <div class="my-turn-div-1">
+                <div class="my-turn-div p-sm-0">
+                  <img class="my-turn-img mb-2 w-100" src="./images/delta-airlines.jpg" alt="" />
+                  <div>
+                    <p>How Likely Is It to Contract COVID-19 on a Plane?</p>
+                  </div>
+                </div>
+                <div class="my-turn-div p-sm-0">
+                  <img class="my-turn-img mb-2 w-100" src="./images/delta-airlines.jpg" alt="" />
+                  <div>
+                    <p>How Likely Is It to Contract COVID-19 on a Plane?</p>
+                  </div>
+                </div>
+
+                <div class="my-turn-div p-sm-0">
+                  <img class="my-turn-img mb-2 w-100" src="./images/delta-airlines.jpg" alt="" />
+                  <div>
+                    <p>How Likely Is It to Contract COVID-19 on a Plane?</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <!-- More stories  -->
+
+            <div class="more-stories bg-white my-3 py-3 d-sm-none d-md-block">
+              <h3>MORE STORIES</h3>
+              <div class="mb-4 mt-4 d-flex">
+                <img class="mr-3"
+                  src="https://d.newsweek.com/en/full/1623581/cia-china-agent-arrested-hawaii-fbi.jpg?w=95&h=95&f=100037ee3ee8d33007ebdfb6e4e95333"
+                  alt="" />
+                <div>
+                  <h4 class="font-weight-bold text-black">
+                    Thirty States May Reject Trump's Weekly $400 Unemployment
+                    Boost, 18 Accept
+                  </h4>
+                  <p>
+                    So far, two states are rejecting the plan that pays out less
+                    than the $600 provided under the CARES Act passed by Congress
+                    in March.
+                  </p>
+                </div>
+              </div>
+              <div class="mb-4 mt-4 d-flex">
```

Comment:
- optional: The text should wrap around the image in all screen sizes

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/91077439-72083c00-e617-11ea-925e-e97e627effce.png) |![image](https://user-images.githubusercontent.com/61479648/91077477-7e8c9480-e617-11ea-8c29-1143328f22d3.png)  |